troubleshooting.md

```diff
@@ -107,4 +107,25 @@ lstat /home/myusername/~: no such file or directory
 
   * Replace `~` with `$HOME` or the fully specified directory `/home/myusername`.
     * `$ sudo buildah push alpine oci:${HOME}/myalpine:latest`
+
 ---
+### 5) Rootless podman build fails EPERM on NFS:
+
+NFS enforces file creation on different UIDs on the server side and does not understand User Namespace.
+When a container root process like YUM attempts to create a file owned by a different UID, NFS Server denies the creation.
+See [Shortcomings of Rootless Podman](https://github.com/containers/libpod/blob/master/rootless.md)
+
```

Comment:
Libpod has:
https://github.com/containers/libpod/blob/master/rootless.md
```
Does not work on NFS homedirs

    NFS enforces file creation on different UIDs on the server side and does not understand User Namespace.
    When a container root process like YUM attempts to create a file owned by a different UID, NFS Server denies the creation.
```

Reply:
Should I just remove that link, since it is just pasted?